hl7generator/hl7_generator_v1.py

```diff
@@ -1275,9 +1275,44 @@ def generateELR(numoELRs, conditionCode, output_folder):
         )
 
         ## OBX
-        OBX = (
-        f"OBX|"
-        )
+
+        # The OBX segment is used to transmit a single observation or observation
+        # fragment. It represents the smallest indivisible unit of a report. 
+        
+
+        # ------- Observation --------
+
+        # Creating variables for OBX segments
+        OBX_1 = "1"  # Set ID - OBX (Set ID)
+        OBX_2 = "NM"  # Value Type --- C
+        OBX_3 = "{patDiseaseCode}^{patDisease}^LN"  # Observation Identifier --- R
```

Comment:
This is being print as a string between "", like this -- 
OBX|1|NM|{patDiseaseCode}^{patDisease}^LN||||||||R||||||||20240229165
SPM|